### PR TITLE
Added a baseto the drawing

### PR DESCRIPTION
**Pull request by Satoshi-Sh**
# Issue number
#3 
[Your PR must reference an issue. You can reference the issue by using the # character]

# Description
Added a function to draw a base for the Christmas tree.
[Please describe your contribution]

# Checklist

- [ x] I have read and complied with the Code of Conduct and Content Guidelines as appropriate.
- [ x] I have followed the Contributing Guide.
- [x ] I agree to license my contribution under the License in this repository.
- [ x] The program runs successfully after making my changes.

### CODE REVIEW

File: drawfunctions/satoshi_s.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    t.left(90)
+    t.forward(20)   # Base height
+    t.left(90)
+    t.forward(40)
+    t.left(90)
+    t.forward(20)
+    t.end_fill()
+
+    # Close the turtle graphics window
+    turtle.done()
```
Comment (dominicduffin1) [inline drawfunctions/satoshi_s.py]: Can you please remove the `turtle.done()` function call?

```suggestion
    
```

File: drawfunctions/satoshi_s.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import turtle
+
+
+def base():
+    t = turtle.Turtle()
+    t.color("brown")
+
```
Comment (dominicduffin1) [inline drawfunctions/satoshi_s.py:7]: Could you please add a `hideturtle` call before starting the drawing? This helps with performance.

```suggestion
    t.hideturtle()
```

**Review by dominicduffin1 — CHANGES_REQUESTED** [review]
Thank you @Satoshi-Sh! Looks good, just a couple of minor suggestions!

File: drawfunctions/satoshi_s.py (hunk 2 above)
Comment (Satoshi-Sh) [reply drawfunctions/satoshi_s.py:7]: Updated those changes, @dominicduffin1

**Review by dominicduffin1 — APPROVED** [review]

**Comment by dominicduffin1** [comment]
Thank you @Satoshi-Sh, merging now!

**Pull request merged**